src/views/Home.jsx

```diff
@@ -1,8 +1,29 @@
 import './Home.css';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import { SingleList } from '../components';
+import { createList } from '../api/firebase';
+
+export function Home({ data, setListPath, user }) {
+	const [shoppingListName, setShoppingListName] = useState('');
+	const navigate = useNavigate(); // useNavigate doc suggests using redirect
```

Comment:
According to some Googling, `redirect` is for use in route loaders and actions, not directly w/in components. So useNavigate is probably the best route (hehe) to go.